tests/Services/StringManagerTest.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+
+namespace Services;
+
+use stdClass;
```

Comment:
no need to use **stdClass** because stdClass is part of PHP global namespace.

Reply:
I know, if you don't include it in your code, you should see something like this /stdClass() in our code, it would be the best practice to treat the stdClass like any other class.